### DISCUSSION TITLE
MC-Server start

### DISCUSSION THREAD

**Discussion by adrianschw** [GameServerManagers/LinuxGSM#3696] [General]
Hi there,

I can no longer start my MC server.

Version: 1.18
Java: 16.0.1

**Comment by adrianschw** [comment]
I have this Error:

`[ FAIL ] Starting mcserver: Unable to start xyz`

**Comment by dgibbs64** [comment]
You need to update to Java 17. This is the new Minecraft requirement. I will be releasing an update to LinuxGSM with the changes soon :)

**Reply by adrianschw** [reply]
Okay thx :)

**Reply by vsvm-v** [reply]
Does this mean that until you release the update, LinuxGSM Minecraft servers wont work?
Trying to do a fresh install with no success.

**Reply by PlayMTL** [reply]
No, you don't need to wait for LinuxGSM update. You can install an other java version on your own. The best way is to google you distro + install java 17. e.g.: "ubuntu 20.04 install java 17".
